src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/OrchardCore.Apis.GraphQL.csproj

```diff
@@ -5,7 +5,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="Assets\css\graphiql-orchard.css" />
+    <Compile Remove="Assets\**" />
+    <EmbeddedResource Remove="Assets\**" />
```

Comment:
We still don't need to embed Assets. If this is done in another module file an issue.

Reply:
Scratch taht, this is not what you are doing, Good!!!